html_css/grid-lessons/advanced_grid_properties.md

```diff
@@ -85,7 +85,7 @@ Check it out for yourself!
 
 ### Fractional units
 
-Now that we know how to quickly create many grid tracks, it's time to learn how to start making them _dynamic_. Dynamic, in this context, simply means "flexible" or "responsive in some way." The opposite of dynamic is _static_, or fixed at a certain defined height, like `150px`, which we used in the setup of this sample grid.
+Now that we know how to quickly create many grid tracks, it's time to learn how to start making them _dynamic_. Dynamic, in this context, simply means "flexible" or "responsive in some way". The opposite of dynamic is _static_, or fixed at a certain defined height, like `150px`, which we used in the setup of this sample grid.
```

Comment:
This period just needs to stay inside the quotation marks.
```suggestion
Now that we know how to quickly create many grid tracks, it's time to learn how to start making them _dynamic_. Dynamic, in this context, simply means "flexible" or "responsive in some way." The opposite of dynamic is _static_, or fixed at a certain defined height, like `150px`, which we used in the setup of this sample grid.
```

Reply:
i thought it was an punctuation error! was i wrong about this?

btw, i'll accept your suggested commit regardless, just want to know, thanks :)

Reply:
No worries! Our curriculum follows US English which requires periods inside the quotations "just like this."

Hope that helps and thanks again for the PR!